src/runtime/driver/handle.rs

```diff
@@ -0,0 +1,272 @@
+use io_uring::squeue;
+use std::cell::RefCell;
+use std::io;
+use std::os::unix::io::{AsRawFd, RawFd};
+use std::rc::{Rc, Weak};
+use std::task::{Context, Poll};
+
+use crate::buf::fixed::FixedBuffers;
+use crate::runtime::driver::op::{Completable, Lifecycle, MultiCQEFuture, Op, Updateable};
+use crate::runtime::driver::Driver;
+
+#[derive(Clone)]
+pub(crate) struct Handle {
+    inner: Rc<RefCell<Driver>>,
+}
+
+#[derive(Clone)]
+pub(crate) struct WeakHandle {
+    inner: Weak<RefCell<Driver>>,
+}
+
+impl Handle {
+    pub(crate) fn new(b: &crate::Builder) -> io::Result<Self> {
+        Ok(Self {
+            inner: Rc::new(RefCell::new(Driver::new(b)?)),
+        })
+    }
+
+    pub(crate) fn tick(&self) {
+        self.inner.borrow_mut().tick()
+    }
+
+    pub(crate) fn flush(&self) -> io::Result<usize> {
+        self.inner.borrow_mut().uring.submit()
+    }
+
+    pub(crate) fn register_buffers(
```

Comment:
I'm going to move a bunch of these methods into the Driver and do delegated calls here, but in another PR.